rc/filetype/nginx.kak

```diff
@@ -0,0 +1,30 @@
+hook global BufCreate .*\.(nginx) %{
+    set-option buffer filetype nginx
```

Comment:
I don't think `*.nginx` files are a thing

Reply:
I couldn't think of anything else here. You'e right. I'll add `params`, remove `nginx` and replace `*` by `+`. I guess that'll do it for now. I guess.

Reply:
@krobelus  if you were to tell me now that say, the fastcgi_params and uwscgi_params are considered to be configuration files and as such such be included on the `conf.kak` instead, I would disagree. I mean. What would be the point of having those *files* there in the first place?

Reply:
I'll go over it again @krobelus. On a system with gcc 11.2 works fine, whereas on 12.2 doesn't.